lib/cli/index.js

```diff
@@ -131,7 +132,10 @@ function parseReporterOptions(options) {
     return options.reporter.map(function(name) {
         return {
             name,
-            path: options[`${name}ReporterPath`]
+            path: options[`${name}ReporterPath`],
+            options: {
+                failedOnly: options.htmlFailedOnly
+            }
```

Comment:
I suggest pass `path` option in an options object. This will reduce the number of parameters passed to the function.